modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js

```diff
@@ -0,0 +1,304 @@
+ * details.
+ */
+
+import ClayIcon from '@clayui/icon';
+import ClayLabel from '@clayui/label';
+import ClayLink from '@clayui/link';
+import ClaySticker from '@clayui/sticker';
+import ClayTabs from '@clayui/tabs';
+import React, {useMemo, useState} from 'react';
+
+import Sidebar from './Sidebar';
+
+const formatDate = (date, languageTag) => {
+	return (
+		date &&
+		languageTag &&
+		Intl.DateTimeFormat(languageTag, {
+			day: 'numeric',
+			hour: 'numeric',
+			hour12: true,
+			minute: 'numeric',
+			month: 'short',
+			year: 'numeric',
+		}).format(new Date(date))
+	);
+};
+
+const SidebarPanelInfoView = ({
+	categories = [],
+	classPK,
+	createDate,
+	data = {},
+	languageTag = 'en',
+	modifiedDate,
+	subType,
+	tags = [],
+	title,
+	userName,
+	userPortraitURL,
+	versions = [],
+	viewURLs = [],
+}) => {
+	const [activeTabKeyValue, setActiveTabKeyValue] = useState(0);
+
+	const formattedCreateDate = useMemo(() => {
+		return formatDate(createDate, languageTag);
+	}, [createDate, languageTag]);
+
+	const formattedDisplayDate = useMemo(() => {
+		return formatDate(data['display-date']?.value, languageTag);
+	}, [data, languageTag]);
+
+	const formattedExpirationDate = useMemo(() => {
+		return formatDate(data['expiration-date']?.value, languageTag);
+	}, [data, languageTag]);
+
+	const formattedModifiedDate = useMemo(() => {
+		return formatDate(modifiedDate, languageTag);
+	}, [modifiedDate, languageTag]);
+
+	const formattedReviewDate = useMemo(() => {
+		return formatDate(data['review-date']?.value, languageTag);
+	}, [data, languageTag]);
+
+	const sortedViewURLS = useMemo(() => {
+		return viewURLs
+			.sort((a, b) => {
+				if (a.languageId < b.languageId) {
+					return -1;
+				}
+
+				if (a.languageId > b.languageId) {
+					return 1;
+				}
+
+				return 0;
+			})
+			.sort((a) => {
+				if (a.default) {
+					return -1;
+				}
+
+				return 0;
+			});
+	}, [viewURLs]);
+
+	return (
+		<>
+			<Sidebar.Header subtitle={subType} title={title}>
+				{versions.map((version) => (
+					<div key={version}>
+						<ClayLabel displayType="info">
+							{`${Liferay.Language.get('version')} ${
+								version.version
+							}`}
+						</ClayLabel>
+
+						<ClayLabel displayType={version.statusStyle}>
+							{version.statusLabel}
+						</ClayLabel>
+					</div>
+				))}
+			</Sidebar.Header>
+
+			<Sidebar.Body>
+				<ClayTabs modern>
+					<ClayTabs.Item
+						active={activeTabKeyValue === 0}
+						innerProps={{
+							'aria-controls': 'tabpanel-0',
+						}}
+						onClick={() => setActiveTabKeyValue(0)}
+					>
+						{Liferay.Language.get('details')}
+					</ClayTabs.Item>
+				</ClayTabs>
+
+				<ClayTabs.Content activeIndex={activeTabKeyValue} fade>
+					<ClayTabs.TabPane aria-labelledby="tab-1" className="mt-3">
+						<div className="mb-4 sidebar-dl sidebar-section">
+							<ClaySticker className="sticker-user-icon">
+								<div className="sticker-overlay">
```

Comment:
ClaySticker already wraps content in `sticker-overlay`, is this additional wrapper needed?

Also, this makes me think we should probably add img support for sticker internally for clay

Reply:
Nice catch! I just copied the example from [docs](https://next.clayui.com/docs/components/sticker.html#user-icon), so maybe we should update them.

> Also, this makes me think we should probably add img support for sticker internally for clay

Yeah, since we provide examples and markup for it add img support might be a good idea

Reply:
Added an issue to fix docs https://github.com/liferay/clay/issues/3541